cuda.mk

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2025. ALL RIGHTS RESERVED.
+# See file LICENSE for terms.
+#
+
+SUFFIXES = .cu
```

Comment:
Does it make sense to check this on `test/apps/test_cuda_get_symbol_address.cu`?